portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -229,15 +227,65 @@ function allowUserSubmit() {
       });
 }
 
-/* inserts a functioning searchbar into the navigation bar of a page. */
-function insertSearch() {
+/**
+ * Gets the other html pages as documents and invokes method to insert Search
+ * bar
+ */
+async function loadSearch() {
+  const otherDocs =
+      await Promise.all([getHTML('/about.html'), getHTML('/statistics.html')]);
+
+  let otherElements = [];
+  otherDocs.forEach((doc) => {
+    otherElements = [...otherElements, ...Array.from(doc.body.childNodes)];
```

Comment:
Just out of my own curiosity, what is the functionality of the ellipsis here?

Reply:
+1

Reply:
It is just the format to combine arrays.